README.md

```diff
@@ -515,6 +515,13 @@ class Response < Hashie::Mash
 end
 ```
 
+The default is to disable logging for all methods that conflict. If you would only like to disable the logging for specific methods, you can include an array of method keys:
+```ruby
+class Response < Hashie::Mash
+  disable_warnings :zip
+end
+```
+
```

Comment:
Let's describe what happens when you specify multiple options and/or call `disable_warnings` multiple times?

Reply:
This is sort of covered by the `'allows grandchild classes to be more strict than child'` test, but I agree I can be more explicit with the description and test scenario

Reply:
I mean the doc says "you can include an array of method keys", but only shows one. For example:

---

The default is to disable logging for all methods that conflict. If you would only like to disable the logging for specific methods, you can include an array of method keys:

```ruby
class Response < Hashie::Mash
  disable_warnings :zip, :zap
end
```
This behavior is cumulative. The examples above and below behave identically.

```ruby
class Response < Hashie::Mash
  disable_warnings :zip
  disable_warnings :zap
end
```

Reply:
I like dB's documentation example here. Being explicit about the different ways you could use it is 🙌